keyboards/hotdox76v2/keymaps/ifohancroft/README.md

```diff
@@ -0,0 +1,20 @@
+![IFo Hancroft HotDox76v2 Layout Image](https://i.imgur.com/bSWyBCV.png)
```

Comment:
This renders it at a smaller, but still hi-res version that is more appropriate for github markdown rendering.
```suggestion
![IFo Hancroft HotDox76v2 Layout Image](https://i.imgur.com/bSWyBCVh.png)
```

Reply:
Good call!